.github/workflows/ci.yml

```
@@ -41,10 +41,10 @@ jobs:
           args: --all --check
 
       - name: cargo clippy
-        uses: actions-rs/clippy-check@v1
+        uses: actions-rs/cargo@v1
         with:
+          command: clippy
           args: --all --all-features -- -A clippy::incorrect_clone_impl_on_copy_type -A clippy::arc_with_non_send_sync
```

this we can remove now #4574
```suggestion
          args: --all --all-features -- -A clippy::arc_with_non_send_sync
```

---

removed both of them, CI passes